sos/report/plugins/rhc.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2023 Red Hat, Inc., Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class Rhc(Plugin, RedHatPlugin):
+
+    """
+    This plugin collects configuration of the RHC package.
+    """
+    short_desc = 'Red Hat client for remote host configured services'
+
+    plugin_name = "rhc"
+    packages = ("rhc", )
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/rhc/*",
```

Comment:
No secrets we need to obfuscate in any of the files here?

Reply:
I haven't found references to secrets in the documentation. There's an entry for user and password in the command line tool, but nothing for /etc/rhc/*. I've asked to the original bug reporter.

Reply:
Per https://github.com/theforeman/foreman-operations-collection/blob/develop/roles/cloud_connector/templates/foreman_rh_cloud.toml.j2 , anything behind `FORWARDER_PASSWORD=` is a password. Maybe not quote marks, but we can obfuscate also them. (and maybe I would use a safe variant with optional blank spaces around `=`, just to be sure).